hack/sync-upstream-rbac.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+
+curl -L https://github.com/mikefarah/yq/releases/download/v4.22.1/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
```

Comment:
Could you download this to `bin` in the root of the repo and also only download if the file does not exist. It will make this step a little faster.